README.md

```diff
@@ -1,28 +1,109 @@
 # Market Maker Tools
 
 ## Arby
-A market making bot that allows its users to easily arbitrage between OpenDEX and centralized exchanges.
+Arby is a market maker bot that allows its users to easily arbitrage between [OpenDEX](http://opendex.network/) and centralized exchanges. Users with spare capital are able to "earn interest" on their assets by providing liquidity to the OpenDEX network.
 
-Arby looks for arbitrage opportunities on the OpenDEX network. It will enter active orders with the configured margin to the orderbook and updates them as soon as the price changes.
+The purpose of Arby is to pull liquidity from centralized exchanges into OpenDEX while incentivizing liquidity providers (market makers) financially.
+
+### How it works
+Arby looks for arbitrage opportunities on the OpenDEX network. It will issue orders with the configured margin to the orderbook and update them as soon as the price changes.
+
+Trades on OpenDEX are settled in seconds. This creates a potential arbitrage opportunity between centralized exchanges and OpenDEX.
```

Comment:
```suggestion
Trades on OpenDEX are settled in seconds. This allows arby to execute the counter trade on the connected centralized exchange right after an order is settled on OpenDEX. Since the order on OpenDEX was issued with an additional margin, the trade on the centralized exchange can almost always be settled for a significantly better price. The price difference between the two trades is the profit (the "interest") arby generates.
```